dictionaries/shell/src/miscellaneous-words.txt

```diff
@@ -13,3 +13,4 @@ posix
 privileged
 punct
 xdigit
+noninteractive
```

Comment:
Is it better to move to `dictionaries/docker/src/docker-words.txt`?

Reply:
I can be in both places if it applies.